viral_seq/tests/test_workflow.py

```diff
@@ -1101,12 +1146,31 @@ def test_pearson_aggregation():
         random_state=random_state,
     )
 
+    feature_rank = feature_count["Features"]
     pearson_rank = feature_count["Pearson R"]
-    pearson_rank_exp = [
-        0.9625443164192421,
-        -0.9455651466704694,
-        0.8684269994231254,
-        0.700139321523472,
-    ]
-    # check first four pearson values, numbers have tendency to vary slightly based on dependency versions
-    assert_allclose(pearson_rank[: len(pearson_rank_exp)], pearson_rank_exp, rtol=6e-6)
+    # expected property of the test is ranking kmer_0 > kmer_1 > kmer_3
```

Comment:
this doesn't mesh with the discussion at https://lisdi-git.lanl.gov/treddy/ldrd_virus_work/-/merge_requests/88#note_106024, which indicates:

> it should remain enforceable that this order is always true in the top 4: [`kmer_0` > `kmer_2` > `kmer_3`]

> `kmer_1` is very close in absolute value of correlation to `kmer_0`, so depending on exact shuffling I can see why its magnitude in the target branch actually exceeds `kmer_0`.

In any case, looks like 2 and 3 swapping sometimes when I change seeds, so maybe there is a bug to work out.